lib/sass_spec/test.rb

```diff
@@ -51,6 +53,8 @@ def handle_expected_error_message!(test_case, options)
   expected_error_msg = _extract_error_message(test_case.expected_error, options)
 
   if expected_error_msg != error_msg
+    return false if options[:probe_todo] && test_case.todo?
+    return false if options[:probe_todo] && test_case.warning_todo?
```

Comment:
I don't understand why you want this to pass when failing. I thought the goal was to find tests that were passing.